src/dataflow/utilities/export.ts

```diff
@@ -0,0 +1,32 @@
+import { DataSequence } from "../components/dataflow-program-graph";
+
+export const exportCSV = (dataSequences: DataSequence[]) => {
+  if (!dataSequences) {
+    return;
+  }
+  if (dataSequences.length > 0) {
+    const headers: string = "timestamp,name,units,value";
+    const rows: string[] = [];
+    dataSequences.forEach((sequence) => {
+      sequence.data.forEach((d) => {
+        // x is the timestamp, y is the value
+        const row: string[] = ["" + d.x, sequence.name, sequence.units, "" + d.y];
+        rows.push(row.join(","));
+      });
+    });
+    const allRows = headers + "\n" + rows.join("\n");
+    const csvFilename = "dataflow-export-" + Date.now() + ".csv";
+    const csvBlob = new Blob([allRows], {type: "text/csv;charset=utf-8;"});
+    if (navigator.msSaveBlob) {
```

Comment:
Comment would be useful here. I'm guessing this is some sort of IE/Edge hack? I suspect we've got other IE-incompatible code elsewhere, so there's some question as to whether we should make the effort.

Reply:
Good point - I pulled that functionality in as per Matt's suggestion from Dataflow 2.0 code, so didn't give it too much thought. I'll look into it a bit and see if I can clarify or adjust if necessary.

Reply:
In that case, a comment mentioning that it was cribbed from the Dataflow 2.0 code might be useful/sufficient.